helper/certutil/helpers.go

```diff
@@ -0,0 +1,167 @@
+package certutil
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/sha1"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/mitchellh/mapstructure"
+)
+
+// GetOctalFormatted returns the byte buffer formatted in octal with
+// the specified separator between bytes.
+func GetOctalFormatted(buf []byte, sep string) string {
```

Comment:
It would be great to get the tests in this package separately. This way we can refactor each layer of the code without trying to remember what package tests what other packages.